internal/godocfx/testdata/golden/index.yml

```diff
@@ -88,19 +88,23 @@ items:
     you read it. Here is how to express that:</p>\n<pre class=\"prettyprint\"><code>w
     = obj.If(storage.Conditions{GenerationMatch: objAttrs.Generation}).NewWriter(ctx)\n//
     Proceed with writing as above.\n</code></pre><h3>Signed URLs</h3>\n<p>You can
-    obtain a URL that lets anyone read or write an object for a limited time.\nYou
-    don't need to create a client to do this. See the documentation of\nSignedURL
+    obtain a URL that lets anyone read or write an object for a limited time.\nSigning
```

Comment:
Didn't realize that this pulled in the docs from storage... interesting.

Reply:
Me neither!